tests/api/conftest.py

```diff
@@ -56,6 +53,9 @@ def app_config(app_config):
         "RATELIMIT_AUTHENTICATED_USER": "1000 per minute",
         "CIRCULATION_TRANSACTION_USER_VALIDATOR": lambda x: True,
         "EXTEND_LOANS_LOCATION_UPDATED": False,
+        "SQLALCHEMY_DATABASE_URI":
+            "postgresql+psycopg2://invenio:invenio@localhost/ils",
```

Comment:
This is not needed normally, it is already set [here](https://github.com/inveniosoftware/docker-services-cli/blob/347bce92f8abfc07b58e54850d8a38cf1c89596d/docker_services_cli/config.py#L57) to use the test docker services [here](https://github.com/inveniosoftware/docker-services-cli/blob/347bce92f8abfc07b58e54850d8a38cf1c89596d/docker_services_cli/config.py#L52).